Makefile

```diff
@@ -124,13 +124,13 @@ crossplane:
 	@cp -a $(WORK_DIR)/crossplane/cluster/crds/* $(CRDS_DIR)
 	@$(OK) Crossplane chart has been fetched
 
-generate-chart: crossplane
+helm.prepare.universal-crossplane: crossplane
```

Comment:
@negz the helm linter error you were getting wasn't related to your local setup. Apparently, `helm.lint` target runs `helm.prepare.$(1)` instead of `helm.prepare`. So, we have to override that one.